.github/workflows/pr-tests-code.yml

```diff
@@ -0,0 +1,60 @@
+name: Run code tests
+
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+    paths: 
+      - src/**
+
+jobs:
```

Comment:
It seems to me that both steps are equivalent and can be combined into one with `matrix` tag like in this [example](https://github.com/adamralph/minver/blob/master/.github/workflows/ci.yml#L15)

Reply:
I think there was a difference previously when `actions/setup-dotnet` would not load multiple versions of the .NET SDK simultaneously, and I had a hack in here to fix that.  The script could probably be consolidated with the matrix tag now.

Reply:
There was one difference: I didn't bother having the windows run build the release version.  I'm sure if it builds on ubuntu, it will build on windows.  And windows is so much slower to run workflows.  So I just added a conditional.

Reply:
How much slower?

Reply:
Try it and see. It’s like 2x-3x slower